Kinvey-Xamarin/Store/DataStore.cs

```diff
@@ -101,6 +101,9 @@ public void SetCustomRequestProperty (string key, JObject value)
 			this.customRequestProperties.Add (key, value);
 		}
 
+		KinveyRealtimeDelegate<T> RealtimeCallback { get; set; }
```

Comment:
It'll be nicer to use either the term `delegate` or `callback`, instead of both. I think we use `delegate` more widely.